test/integration/psetv2.spec.ts

```diff
@@ -1293,6 +1293,68 @@ describe('liquidjs-lib (transactions with psetv2)', () => {
       await regtestUtils.broadcast(tx.toHex());
     },
   );
+
+  it('can create (and broadcast via 3PBP) a confidential timelocked transaction', async () => {
+    const alice = createPayment('p2wpkh', undefined, undefined, true);
+    const bob = createPayment('p2wpkh', undefined, undefined, true);
+    const aliceInputData = await getInputData(alice.payment, true, 'noredeem');
+    const bobInputData = await getInputData(bob.payment, true, 'noredeem');
+    const inputs = [aliceInputData, bobInputData].map(({ hash, index }) => {
+      const txid: string = Buffer.from(hash).reverse().toString('hex');
+      return new CreatorInput(txid, index, undefined, 10);
+    });
+    const outputs = [
+      new CreatorOutput(
+        lbtc,
+        2_0000_0000 - 2000,
+        alice.payment.output,
+        alice.payment.blindkey,
+        0,
```

Comment:
can we try setting the locktime to some other value like `1000`?

Reply:
can we try setting the locktime to some other value like 1000?

```suggestion
        1000,
```